sdk/main/src/ED25519PublicKey.cc

```diff
@@ -42,13 +43,20 @@ namespace
  */
 [[nodiscard]] internal::OpenSSLUtils::EVP_PKEY bytesToPKEY(std::vector<std::byte> bytes)
 {
+  std::vector<std::byte> buildBytes = bytes;
```

Comment:
Same as above, I don't think you need another copy of `bytes` here, you can just use `bytes` in the function.